src/chain/CacheManager.sol

```diff
@@ -134,11 +195,24 @@ contract CacheManager {
         return cacheSize - queueSize;
     }
 
-    /// Evicts entries until enough space exists in the cache, reverting if payment is insufficient.
-    /// Returns the bid and the index to use for insertion.
+    function _calcDecay() internal view returns (uint256) {
```

Comment:
Does Solidity really require `view` over `pure` for the timestamp?

Reply:
yes